modules/mod_speasyimagegallery/mod_speasyimagegallery.php

```diff
@@ -26,7 +26,8 @@
 if($layout == 'albums') {
   $albums = ModSpeasyimagegalleryHelper::getAlbumList($params);
 } else {
-  $images = ModSpeasyimagegalleryHelper::getImages($params);
+  $images           = ModSpeasyimagegalleryHelper::getImages($params);
+  $albumDescription = (!is_null($images) && !empty($images)) ? ModSpeasyimagegalleryHelper::getAlbumDescription($params): null;
```

Comment:
spaces in between colon " : "
```suggestion
  $albumDescription = (!is_null($images) && !empty($images)) ? ModSpeasyimagegalleryHelper::getAlbumDescription($params) : null;
```